apps/web/package.json

```diff
@@ -56,6 +57,9 @@
   "devDependencies": {
     "@babel/core": "^7.17.9",
     "@babel/preset-env": "^7.16.11",
+    "@faker-js/faker": "6.2.0",
+    "@pins/api": "file:../api",
+    "@pins/web": "file:.",
```

Comment:
Why do we need this?

Reply:
So, by default, npm workspaces will be symlinked via node_modules to allow for importing from them like normal. This is why we can import from @pins/platform and @pins/express etc.

However, because the web and api are called "web" and "api" respectively, these are symlinked into node-modules under those names, without the @pins namespace. In order to continue the convention of @pins/*, there was two options:

1. Rename the "web" and "api" package name to "@pins/web" and "@pins/api", or
2. Alias them so that they can retain their names but still be accessed by the @pins namespace.

I tried (1) first, but it meant that the npm scripts then needed the new names (i.e. npm run dev --workspace=@pins/web etc), which seemed less convenient. So, this second way creates a symlink by pointing to a relative folder.

As to why do this at all, it's to avoid having to form long relative path names from the testing folder that are pathed across packages. If we want to use a function/type from the api for testing purposes, we can treat the api like a package and share things via an index.js in its root.

Reply:
I see. I still want to do the first option. Adding @pins/ is not that big of a deal, and can be fixed by simply running `npm run dev` in root, and turbo will take care of the entire thing anyways running both of them with log grouping and all that stuff.

Adding the index file and the reference like this is more confusing than it help IMHO. So please rename the packages.

Reply:
Ok – are we sure this isn't going to break anything in the pipelines by renaming the app packages like this?

Reply:
I've committed this change.